tfidf-benchmark/NLP-pipeline-cuML.ipynb

```diff
@@ -0,0 +1,4769 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #50.        print(hashing_vectorized.shape)
</pre><p>You are computing shape at end of every step. This is the reason for 2x the time. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Removed this. However while it is not 2x anymore, the latency is still larger.

Reply:
<p>Can you share what the errors that you see <code>intermittently</code>  ? </p>

Reply:
<p>Lets pair on this tomorrow. Something seems to be wrong with the large time. </p>